Source/WebKit/WebProcess/Model/ModelProcessModelPlayer.cpp

```diff
@@ -154,6 +155,13 @@ std::optional<std::unique_ptr<WebCore::ModelPlayerTransformState>> ModelProcessM
 void ModelProcessModelPlayer::load(WebCore::Model& model, WebCore::LayoutSize size)
 {
     RELEASE_LOG(ModelElement, "%p - ModelProcessModelPlayer load model id=%" PRIu64, this, m_id.toUInt64());
+
+    if (!WebCore::MIMETypeRegistry::isUSDMIMEType(model.mimeType())) {
+        RELEASE_LOG(ModelElement, "%p - ModelProcessModelPlayer::load: Found unexpected model mimetype: %s", this, model.mimeType().utf8().data());
+        if (m_client)
```

Comment:
It would be good (not in this patch) to make ModelPlayerClient ref counted so that we can do:
`if (RefPtr client = m_client.get())`